sources/packages/backend/libs/integrations/src/atbc-integration/atbc-integration.processing.service.ts

```diff
@@ -56,52 +61,80 @@ export class ATBCIntegrationProcessingService {
   }
 
   /**
-   * Process all the pending PD requests applied by students.
+   * Process all the pending disability requests applied by students.
    */
-  async processPendingPDRequests(): Promise<ProcessPDRequestSummary> {
-    // Students who applied for PD and waiting for confirmation.
-    const students = await this.studentService.getStudentsAppliedForPD();
-    let updatedPDstatusCount = 0;
-
-    if (students.length) {
-      this.logger.log(`Processing PD requests for ${students.length} students`);
+  async processAppliedDisabilityRequests(): Promise<ProcessSummaryResult> {
+    const processSummaryResult: ProcessSummaryResult =
+      new ProcessSummaryResult();
+    // Students who applied for disability status and waiting for confirmation.
+    const studentDisabilityUpdates =
+      await this.atbcService.getStudentDisabilityStatusUpdatesByDate();
 
+    const studentsToUpdate: StudentDisabilityStatusDetail[] =
+      studentDisabilityUpdates.map((student) => ({
+        sin: student.SIN,
+        lastName: student.APP_LAST_NAME,
+        birthDate: getDateOnlyFromFormat(student.BIRTH_DTE, ATBC_DATE_FORMAT),
+        disabilityStatus: student.D8Y_TYPE as DisabilityStatus,
+        disabilityStatusUpdatedDate: getDateOnlyFromFormat(
+          student.D8Y_DTE,
+          ATBC_DATE_FORMAT,
+        ),
+      }));
+    let updatedDisabilityStatusCount = 0;
+    this.logger.log(
+      `Processing disability status requests for ${studentsToUpdate.length} students`,
```

Comment:
As a suggestion, not a show stopper or part of your PR, to handle, both the variety of logs generated below:
Processing disability status requests for 1 student
Processing disability status requests for 2 students

It would be better to have the student(s) in the logger.